client/ayon_core/hosts/houdini/api/lib.py

```diff
@@ -391,12 +402,14 @@ def imprint(node, data, update=False):
 
     # Add new parm templates
     if new_parm_templates:
-        parm_folder = parm_group.findFolder("Extra")
+        parm_folder = (
+            parm_group.findFolder("AYON") or parm_group.findFolder("Extra")
+        )
```

Comment:
Interesting. I had originally thought of making the folder name an argument to the `imprint` function because it can be quite useful for other usages of the `imprint` library function.

Anyway, I can see how this might be worth it from a backwards compatibility standpoint to do it like this.